### ISSUE TITLE
Ensure contents are loaded 

### ISSUE THREAD

**Issue by mooyoul** [apex/capture#3]
[Currently, PhantomJS wont wait for Content are loaded](https://github.com/apex/capture/blob/master/convert.js#L33-L37) (e.g. DOMContentLoaded  / Load / etc.)

It would be nice if we ensure contents are loaded :)

**Comment by tj** [comment]
Yeah for sure, not sure if phantom has a good mechanism for that ATM, but it would be nice to have. In lots of cases (SPAs etc) there isn't really a final loaded state, so you still need a timeout there

**Comment by tj** [comment]
Ah apparently it does, just doesn't use it in the examples for whatever reason
